### ISSUE TITLE
Need help for training a detector

### ISSUE THREAD

**Issue by YAOYI626** [KaihuaTang/Scene-Graph-Benchmark.pytorch#16]
## ❓ Questions and Help
Hi Kaihua!

Following the cmd in cmd.cache to train a detector based on VGG16 but I always got "Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 5.940911144672375e-213". 

As you said, the cost of training a detector is some unacceptable. Compared to model based on X_101 backbone, it is harder to find a one based on VGG16, so would you like to upload the pretain weight of FasterRCNN based on VGG16?  It may be a little difficult, but absolutely will be a super precious resource.

Best wishes.

**Comment by KaihuaTang** [comment]
In this work，we didn't use VGG16 backbone. A previously popular codebase https://github.com/rowanz/neural-motifs did so. If you prefer VGG16, maybe you can check their release detector.  The cmds in cmd.cache are expired for a long time, but based on your problem, I guess using a smaller learning rate may solve the gradient overflow problem.

**Comment by YAOYI626** [comment]
got it. Thanks!
